frontend/public/components/api-explorer.tsx

```diff
@@ -666,15 +680,15 @@ const APIResourcePage_ = ({
   }
 
   if (flags[FLAGS.OPENSHIFT]) {
-    pages.push({
-      href: 'access',
-      name: 'Access Review',
-      component: APIResourceAccessReview,
-    });
+    if (canCreate) {
```

Comment:
This should be

```suggestion
    if (flags[FLAGS.OPENSHIFT] && canCreate) {
```

as one `if` statement